### ISSUE TITLE
Breadcrumbs

### ISSUE THREAD

**Issue by aaronshekey** [StackExchange/Stacks#234]
In an effort to further orient our users, certain pages can benefit from breadcrumbs. We ought to have a component for this.

---
cc: @ludwiczakpawel

**Comment by ludwiczakpawel** [comment]
Hey, I played a bit with some ideas here: https://www.figma.com/file/IqQEATwAnr1SEEKAfSadjM62/Breadcrumbs?node-id=0%3A1

Nothing really set in stone and I wonder how that would look like with real context (I've only checked _some_ views). Also, using breadcrumbs would be dependent on what page you're currently on. Because it doesn't make sense to have it everywhere I believe. 

AFAIK @TedGoas also had some ideas for breadcrumbs..?
